### DISCUSSION TITLE
filterDeep return undefined if no children found

### DISCUSSION THREAD

**Discussion by ferreirix** [YuriGor/deepdash#103] [Ideas]
Is it possible to have children as undefined if no children respects the predicate?

For instance, [in this example](https://codepen.io/jpferreira/pen/poEWZrY?editors=0010), can the second object in the array have `children` as `undefined` instead of empty array?

```
{
    "name": "2",
    "bad": true,
    "children": undefined
}
```
vs 
```
{
    "name": "2",
    "bad": true,
    "children": []
}
```

Thank you in advance for your help.

**Comment by YuriGor** [comment]
Hi, no such option exists. 

You can do second filtering like this:
https://codepen.io/yurigor/pen/QWKqBRR?editors=0010

It would be nice to have such option so maybe I will add it, once I will have some time for this.
